community-containers/local-ai/readme.md

```diff
@@ -3,13 +3,7 @@ This container bundles Local AI and auto-configures it for you.
 
 ### Notes
 - Make sure to have enough storage space available. This container alone needs ~7GB storage. Every model that you add to `models.yaml` will of course use additional space which adds up quite fast.
-- After the container was started the first time, you should see a new `nextcloud-aio-local-ai` folder when you open the files app with the default `admin` user. In there you should see a `models.yaml` config file. You can now add models in there. Please refer [here](https://github.com/mudler/LocalAI/blob/master/gallery/index.yaml) where you can get further urls that you can put in there. Afterwards restart all containers from the AIO interface and the models should automatically get downloaded by the local-ai container and activated.
-- Example for content of `models.yaml` (if you add all of them, it takes around 10GB additional space):
-```yaml
-# Stable Diffusion in NCNN with c++, supported txt2img and img2img 
-- url: github:mudler/LocalAI/blob/master/gallery/stablediffusion.yaml
-  name: Stable_diffusion
-```
+- After the container was started the first time, you can open `http://ip.address.of.server:8000` and log in with the api key that you can retrieve when running... 
```

Comment:
the problem is that it uses http and not https... so this needs to be proxied anyway...